packages/rest/src/validation/request-body.validator.ts

```diff
@@ -115,7 +115,7 @@ function getKeyForOptions(options: RequestBodyValidationOptions) {
  * @param globalSchemas - Schema references.
  * @param options - Request body validation options.
  */
-async function validateValueAgainstSchema(
+export async function validateValueAgainstSchema(
```

Comment:
Let's change the reference to `body` with a generic one, such as `value`.

Reply:
done.